docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.7"
```

Comment:
I don't think you need to specify a version for recent versions of docker. This is likely a pattern from an old tutorial.